MetaMask/metamask-extension
# Upgrading storybook from 6.3 to 6.5 fixing shell-quote vulnerability

## Explanation

Currently, there is a [shell-quote vulnerability](https://github.com/advisories/GHSA-g4rg-993r-mgx7) that is effecting many of the storybook `6.3` packages 

Not sure how big of an issues this is because it only effects our development builds. I suppose it could be used to attack our storybook PR builds and Github pages in some way? Nevertheless, it will clean up our yarn audit report(`yarn audit --level critical`) **AND** we get [storybook 6.5 which is full of goodies](https://storybook.js.org/blog/storybook-6-5/) that I am excited to try out 😁 

In order to solve this problem, this pull request upgrades storybook `6.3` => `6.5`. It also removes the "ws" package resolution that was made in https://github.com/MetaMask/metamask-extension/pull/11229 as I no longer believe it is needed.

## More Information

* Fixes #15683
* Related: #15656

## Screenshots/Screencaps
Run `yarn audit --level critical`

### Before

![Screen Shot 2022-08-23 at 2 42 58 PM](https://user-images.githubusercontent.com/8112138/186275409-d034f7e8-b9be-423a-b4be-91b9bf2601ce.png)
![Screen Shot 2022-08-23 at 2 43 17 PM](https://user-images.githubusercontent.com/8112138/186275413-80e1e8b8-c869-47b9-8f10-4660728ca7df.png)
![Screen Shot 2022-08-23 at 2 43 28 PM](https://user-images.githubusercontent.com/8112138/186275415-815d906c-ac08-4d4f-ab8b-d9585abe50ab.png)


### After
![Screen Shot 2022-08-23 at 2 37 09 PM](https://user-images.githubusercontent.com/8112138/186275445-47079722-3bac-4e52-8467-0b920b63c5dd.png)


## Manual Testing Steps
Checking yarn audit report
- Check out this branch
- Run `yarn audit --level critical`
- See reduced number of vulnerabilities compared to `develop`

Checking storybook
- Go to the latest storybook build of this PR
- Check stories are working as expected - controls work, MDX docs work, Prop tables are showing up

## Pre-Merge Checklist


### + If there are functional changes:
